src/plugins/kibana_react/public/field_button/field_button.tsx

```diff
@@ -29,8 +31,11 @@ export interface FieldButtonProps {
   isDraggable?: boolean;
   size?: ButtonSize;
   className?: string;
+  onClick?: () => void;
 }
 
+export type FieldButtonProps = OneOf<Props, 'fieldInfoIcon' | 'fieldAction'>;
```

Comment:
The styles only account for one or the other so I enshrined that in TS as well.

Reply:
`field_button` should support both `fieldAction` and `fieldInfoIcon` like this:

<img width="269" alt="Pasted Graphic 1" src="https://user-images.githubusercontent.com/4016496/89134511-a1290300-d4da-11ea-865e-d268b69f6d50.png">